pages/web/login/index.tsx

```diff
@@ -20,7 +20,7 @@ const LoginNextPage: NextPage = () => {
 
   return (
     <div>
-      <Head title='Login' url='/login' />
+      <Head title='Login' url='/web/login' />
```

Comment:
bu SEO uchun ishlatiladi, bugni fix qilmaydi

Reply:
understood im gonna fix the problem ASAP